FreeTAKServer/controllers/configuration/MainConfig.py

```diff
@@ -2,16 +2,18 @@
 import re
 import yaml
 currentPath = os.path.dirname(os.path.abspath(__file__))
+
 from pathlib import Path
 from string import ascii_letters, digits, punctuation
 from uuid import uuid4
 
 # the version information of the server (recommended to leave as default)
-FTS_VERSION = 'FreeTAKServer-1.9.9.6 Public'
+FTS_VERSION = 'FreeTAKServer-1.9.9.7 Public'
 API_VERSION = '1.9.5'
 # TODO Need to find a better way to determine python version at runtime
 PYTHON_VERSION = 'python3.8'
 USERPATH = '/usr/local/lib/'
+PERSISTENCE_PATH = '/opt/fts/'
 MAINPATH = fr'{USERPATH}{PYTHON_VERSION}/dist-packages/FreeTAKServer'
```

Comment:
I think that `MAINPATH` pretty much goes away at this point. Which is fine by me. I had started a branch to detect what version of Python was in use to auto configure the `MAINPATH`, but with this PR it should not be needed.